packages/experiments/src/components/Announced/Announced.base.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { BaseComponent, DelayedRender, classNamesFunction } from '../../Utilities';
+import { IProcessedStyleSet } from '../../Styling';
+import { IAnnouncedProps, IAnnouncedStyles, IAnnouncedStyleProps } from './Announced.types';
+
+const getClassNames = classNamesFunction<IAnnouncedStyleProps, IAnnouncedStyles>();
+
+export class AnnouncedBase extends BaseComponent<IAnnouncedProps, {}> {
+  private _classNames: IProcessedStyleSet<IAnnouncedStyles>;
+  public render(): JSX.Element {
+    const { message, styles, theme, id } = this.props;
+
+    this._classNames = getClassNames(styles, {
+      theme: theme!
+    });
+
+    return (
+      <div role="status" aria-live="assertive" id={id}>
```

Comment:
I think that we should allow the `aria-live` to be overridden and that it should just default to assertive.